### ISSUE TITLE
[false-negative] quoted column alias detected as keyword

### ISSUE THREAD

**Issue by ancoron** [alanmcruickshank/sqlfluff#344]
The following fails:
```sql
CREATE VIEW v_test AS
SELECT timestamp_ as "timestamp"
FROM t_test;
```
...with:
```
== [db/test.sql] FAIL
L:   2 | P:  19 | L010 | Inconsistent capitalisation of keywords.
```

**Comment by alanmcruickshank** [comment]
I don't think this is a bug, it's picking up on your inconsistent capitalisation of `as` on line 2. I wonder if the description should actually read out the keyword that it's complaining about which might improve UX.

@ancoron - thoughts?

**Comment by ancoron** [comment]
Oh, my bad then. I think if I had read better I would have noticed that at position 19 the `as` keyword begins and not `"timestamp"`.

OTOH, for parsing errors it shows the start of the SQL that it's unable to parse, so having the keyword complained about also shown in addition would certainly help identifying it a bit faster for humans.
